test/lib/ansible_test/_data/requirements/constraints.txt

```diff
@@ -1,7 +1,7 @@
 # do not add a cryptography or pyopenssl constraint to this file, they require special handling, see get_cryptography_requirements in python_requirements.py
 # do not add a coverage constraint to this file, it is handled internally by ansible-test
 packaging < 21.0 ; python_version < '3.6' # packaging 21.0 requires Python 3.6 or newer
-paramiko < 2.9.0 # paramiko 2.9.0+ requires changes to the paramiko_ssh connection plugin to work with older systems
+paramiko >= 2.9.0 # HACK: force paramiko upgrade to test changes against newer versions
```

Comment:
Should this line just be removed now I assume?

Reply:
Yes, now that the tests have passed the line can be removed.

Reply:
removed